addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -637,8 +637,16 @@ odoo.define('website.s_website_form', function (require) {
                 }
 
                 const formData = new FormData(this.$target[0]);
-                const currentValueOfDependency = formData.get(dependencyName);
-                return this._compareTo(comparator, currentValueOfDependency, visibilityCondition, between);
+            	var currentValueOfDependency;
+		if(["contains","!contains"].includes(comparator))
```

Comment:
The code works fine, but you should follow the coding style. For example the code inside the if block is not properly aligned.

You can use const instead of var since it was defined as const previously.
You could use the ternary (conditional) operator to replace the 'if else' statement.

               
                const currentValueOfDependency = ["contains", "!contains"].includes(comparator)
                    ? formData.getAll(dependencyName).join()
                    : formData.get(dependencyName);

Reply:
I have made the changes